scripts/csv_to_json.py

```diff
@@ -45,19 +47,31 @@ def csv_to_json(csvFile, jsonFile):
     All_positions = []
     for key, value in heirarchy_order.items():
         for i in value:
-            All_positions.append(i)
+            All_positions.append(i.capitalize())
 
     with open(csvFile, encoding="utf-8") as f:
         csvReader = list(csv.DictReader(f))
         for position in All_positions:
             for rows in csvReader:
-                key = rows["Position"]
-                print(position, rows["Position"])
-                if position in rows["Position"]:
-                    if key in json_data:
-                        json_data[key].append(rows)
+                key = rows["Position"].capitalize()
+                if position in key:
+                    if key in position_data:
+                        position_data[key].append(rows)
                     else:
-                        json_data[key] = [rows]
+                        position_data[key] = [rows]
+
+    for key in json_data.keys():
+        for i in range(len(json_data[key])):
+            del json_data[key][i]["Position"]
+
+    dept_list = heirarchy_order.keys()
```

Comment:
Might wish to correct the spelling of the word,
`heirarchy` => `hierarchy`.

Reply:
I did make that mistake in a lot of places xD. Made changes as requested.